eel/edge.py

```diff
@@ -1,20 +1,48 @@
-import platform
+import os
 import subprocess as sps
 import sys
 from typing import List
 
 from eel.types import OptionsDictT
 
-name: str = 'Edge'
+name: str = "Edge"
 
 
-def run(_path: str, options: OptionsDictT, start_urls: List[str]) -> None:
-    cmd = 'start microsoft-edge:{}'.format(start_urls[0])
-    sps.Popen(cmd, stdout=sys.stdout, stderr=sys.stderr, stdin=sps.PIPE, shell=True)
+def run(path: str, options: OptionsDictT, start_urls: List[str]) -> None:
+    if path != "edge_legacy":
+        if options["app_mode"]:
+            for url in start_urls:
+                sps.Popen([path, "--app=%s" % url] + options["cmdline_args"], stdout=sps.PIPE, stderr=sps.PIPE, stdin=sps.PIPE)
```

Comment:
For consistency I'd suggest you break the long lines here for the calls to `sps.Popen` here and below, similar to how they are implemented in `chrome.py`?